sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/README.md

```diff
@@ -0,0 +1,108 @@
+# Azure Event Hubs blob storage checkpoint library for .NET
+
+WORDS NEEDED...
```

Comment:
This is intended to be a skeleton;  content will be added in a future PR, including updates to the associated ARM template and other assets.